### DISCUSSION TITLE
Toggle Ingredient/Instruction Placeholder Text in Settings for Manual Recipe Creation:

### DISCUSSION THREAD

**Discussion by PinkyWings** [mealie-recipes/mealie#3757] [Feature Request]
### First Check

- [X] I used the GitHub search to find a similar requests and didn't find it.
- [x] Checked the [tasks tagged](https://github.com/mealie-recipes/mealie/issues?q=is%3Aissue+is%3Aopen+label%3Atask+) issues and verified my feature is not covered

### Please provide a concise description of the problem that would be addressed by this feature.

When a recipe is manually created (not a URL import), the 'Ingredients' and 'Instructions' have a placeholder entry to help a new user to explain how to use the section.
![MealieSnipPlaceholderManualRecipe](https://github.com/mealie-recipes/mealie/assets/95440963/71682d0d-fd25-405e-800c-692539351ee0)
 

### Please provide a concise description of the feature that would resolve your issue.

The idea is to add a setting to toggle the placeholders on and off. If the user is experienced, they can toggle the feature off in settings and they do not have to delete the placeholders every time a manual recipe is created.

### Please consider and list out some caveats or tradeoffs made in your design decision

Makes settings options more complicated

### Additional Information

- [ ] If this is accepted I'm willing to submit a PR to provide this feature
- [ ] If this is accepted I'm willing to help maintain this feature
- [ ] I'm willing to sponsor/pay a developer to do this work

**Comment by michael-genson** [comment]
Ideas for an improved new recipe workflow are being tracked here: https://github.com/mealie-recipes/mealie/issues/2012
